garage/theano/algos/ddpg.py

```diff
@@ -37,11 +35,11 @@ def __init__(self,
                  policy,
                  qf,
                  es,
+                 pool,
```

Comment:
It definitely makes sense to move it out, since we don't want the algorithm to care (much) about the replay buffer type.
A default of None (which produces a SimpleReplayBuffer), sounds reasonable to me. However, I don't think that needs to go in this pull request.